external/hiring.md

```diff
@@ -0,0 +1,40 @@
+# Software Developer Hiring Process
+
+## Purpose
+
+To standardize and organize our hiring process for tech team devs, so we can
+be more inclusive and transparent to candidates.
+
+## Steps
+
+1. Resume review
+    - You can expect several team members to review your resume
+    - Don't stress too much about this, we know getting to know you takes more than a resume.
+2. Phone screen candidate
+    - ~25 minutes with one **Tech Team** member
```

Comment:
I'm starting to feel like the number-of-calls we list may not be realistic.  And that we'll end up with an additional "talk to the tech team" after the phone screen, but before the technical venture.  The 2 touchpoints of "phone screen" + "technical interview" just isn't a ton of time for us to get to know the candidate.

Or maybe I'm just stuck in my old interviewing paradigms.